test/integration/employees_edit_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class EmployeesEditTest < ActionDispatch::IntegrationTest
+
+  def setup
+    @employee = employees(:MyString)
+  end
+ 
+  test "successful edit with friendly forwarding" do
```

Comment:
Keep your comments for tests very specific. This is ambiguous. If I were to say something like, "When X happens, Y results," "When X happens, Y has Z items," or "X renders Y page," it is very clear and direct what is actually being tested.